[2024-03-03T03:45:01.759+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_4 scheduled__2017-03-14T00:00:00+00:00 [queued]>
[2024-03-03T03:45:01.778+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_4 scheduled__2017-03-14T00:00:00+00:00 [queued]>
[2024-03-03T03:45:01.780+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:45:01.808+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_4> on 2017-03-14 00:00:00+00:00
[2024-03-03T03:45:01.818+0000] {standard_task_runner.py:60} INFO - Started process 5768 to run task
[2024-03-03T03:45:01.824+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_4', 'scheduled__2017-03-14T00:00:00+00:00', '--job-id', '2661', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmp96j_lamk']
[2024-03-03T03:45:01.828+0000] {standard_task_runner.py:88} INFO - Job 2661: Subtask get_files_4
[2024-03-03T03:45:01.859+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:45:01.948+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_4 scheduled__2017-03-14T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:45:02.179+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_4' AIRFLOW_CTX_EXECUTION_DATE='2017-03-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2017-03-14T00:00:00+00:00'
[2024-03-03T03:45:03.537+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:45:03.568+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_4, execution_date=20170314T000000, start_date=20240303T034501, end_date=20240303T034503
[2024-03-03T03:45:03.653+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:45:03.722+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
